### PR TITLE
Tracking Plan Audit section in docs

### PR DESCRIPTION
**Pull request by logason**
Adds a section about Tracking Plan Audit to docs

Screenshot 2021-07-16 at 14 20 20

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**/audit/quickstart**](https://docs-a4w08tz5x.avo-dev.app/audit/quickstart) | [**/audit/rules**](https://docs-a4w08tz5x.avo-dev.app/audit/rules) |
|:-:|:-:|
|<a href="https://docs-a4w08tz5x.avo-dev.app/audit/quickstart">Screenshot of /audit/quickstart</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/cMM5NQYQuNvgI9-TkqWNwEeBZ6j_upRJAD-GvUEI7l0D3ZGuNsSGcplNBD00hT-_pKmLxr8hFK7xs8DZlG4DVw.png">(view full size)</a> |<a href="https://docs-a4w08tz5x.avo-dev.app/audit/rules">Screenshot of /audit/rules</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Y2id6OYI5mReyarDkYJFWtfGwJiJOtL1LoB0lmMJAdSVr6i5cZQHxiNviVZoii3huCvMdzVTZz1djM_c2bgFKw.png">(view full size)</a> |



| [**…racking-plan/importing**](https://docs-a4w08tz5x.avo-dev.app/workspace/tracking-plan/importing) |
|:-:|
|<a href="https://docs-a4w08tz5x.avo-dev.app/workspace/tracking-plan/importing">Screenshot of /workspace/tracking-plan/importing</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/wG-JE4P1Uhx2LVqbnEDKP6kVOTGmiWkSQNCWWnuKjf3PYP7XntfHFNOoc61DjTxWemrKE2b0FO7UotAQfqtAqw.png">(view full size)</a> |







Commit 09ad640190ed047a507f916a18f6698d94f10674 (https://docs-a4w08tz5x.avo-dev.app).

**Comment by logason** [comment]
Everything working except for the spellcheck, which I don't know why is failing on me :/ Any ideas @alex-tpom6oh ?

File: .spelling [hunk 1]
```diff
@@ -142,94 +142,24 @@ accessMembers.png
...
-jitpack.io
-logcat
-nullable
-debuggermanager
-avoinspector
-cocoapods
-nsdictionary
-analyticsdebugger
-propname
-camelCase
```
Comment (alex-tpom6oh) [inline .spelling:218]: Hm, that's why it's failing I guess, some words were removed from our custom dictionary

Comment (logason) [reply .spelling:218]: Ahhh I see! I probably messed up the merge with the main branch 😬

File: pages/audit/index.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+layout: 'guide'
+title: 'What is a Tracking Plan audit?'
+#abstract: 'Learn how to use Avo to identify issues in your existing tracking plan'
```
Comment (alex-tpom6oh) [inline pages/audit/index.mdx:4]: Should we remove comments like this?

File: pages/audit/index.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+import Link from '../../components/Link';
+
+The Avo tracking plan audit reviews your event and property definitions and flags issues such as unexpected naming or inconsistent types — helping you keep your tracking plan clean and consistent.
+
+The tracking plan audit monitors every change you make in your Avo Tracking Plan and highlights if new issues arise. You can think about the tracking plan audit like a spell-checker for your tracking plan.
+
+You can get a tracking plan audit for your existing tracking plan today by importing it into Avo. Follow the quickstart guide below to learn how to import your existing tracking plan into Avo and review your first tracking plan audit.
```
Comment (alex-tpom6oh) [inline pages/audit/index.mdx:13]: This article feels a bit repetitive with the same one in the quick start doc and with the `What's next?` section

File: pages/audit/rules.mdx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+layout: 'guide'
+title: 'Avo tracking plan audit rules'
+abstract: 'Learn how to read and fix your tracking plan audit'
+---
+
+import Link from '../../components/Link';
+
+# Audit rules
+
+Following are the static rules Avo tests for in the tracking plan audit. To learn more about how to get started with the tracking plan audit, see <Link passHref href="/audit/quickstart"><a>Quickstart: Audit in 5 minutes</a></Link>.
```
Comment (alex-tpom6oh) [inline pages/audit/rules.mdx:11]: Should Quickstart: Audit in 5 minutes be the actual title of the `audit/quickstart.mdx` doc?

File: pages/audit/rules.mdx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+Checks if any two event names in the tracking plan conflict with each other. For event name to be unique, no other event can have the same name, independent from their case. Example of conflicting event names:
+
+- "App Opened", "app opened", "app_opened", "app-opened"
+- "Add to Cart", "Add to cart", "add_to_cart", "add-to-cart"
+- "Checkout Started", "checkout started", "checkout-started"
+- "Checkout Completed", "checkout completed", "checkout-completed"
+
+##### How to resolve
+
+Pick the casing that matches the one that is dominant one in your workspace and fix the implementation of those that don't fit your standards. Note that changing your existing tracking might cause issues in downstream tools and metrics. Make sure to discuss with the consumers of the data before making such change.
```
Comment (alex-tpom6oh) [inline pages/audit/rules.mdx:34]: Make `Make sure to discuss with the consumers of the data before making such change.` bold? 😀

**Review by alex-tpom6oh — APPROVED** [review]
Looks good, added some thoughts

File: pages/audit/index.mdx (hunk 2 above)
Comment (logason) [reply pages/audit/index.mdx:4]: I think it's fine to keep it in there as a reminder that a subtitle is missing here, and this is how you could add one. Does that make sense?

File: pages/audit/rules.mdx (hunk 4 above)
Comment (logason) [reply pages/audit/rules.mdx:11]: Ah good catch, it was, then I renamed it, will update it here

File: pages/audit/index.mdx (hunk 3 above)
Comment (logason) [reply pages/audit/index.mdx]: Ah good catch, removed this paragraph in 9f31e1575cd0a6d672211a1dbb9ee81c71692de7

File: pages/audit/rules.mdx (hunk 4 above)
Comment (logason) [reply pages/audit/rules.mdx]: Done in 9f31e1575cd0a6d672211a1dbb9ee81c71692de7

File: pages/audit/rules.mdx (hunk 5 above)
Comment (logason) [reply pages/audit/rules.mdx]: Done in 9f31e1575cd0a6d672211a1dbb9ee81c71692de7

**Review by alex-tpom6oh — APPROVED** [review]
LGTM

**Pull request merged**